BYT/View Controllers/FoaasViewController.swift

```diff
@@ -120,6 +124,20 @@ class FoaasViewController: UIViewController, FoaasViewDelegate, FoaasSettingMenu
         }
         self.foaasView.mainTextLabel.text = validFoaas.message
         self.foaasView.subtitleTextLabel.text = validFoaas.subtitle
+
```

Comment:
can you add a short comment on what this dispatch closure is doing? it's not clear to me